resources/analysisTools/copyNumberEstimationWorkflow/phasing.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash/
+
+# Copyright (c) 2017 The ACEseq workflow developers.
+# Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/ACEseqWorkflow/LICENSE.txt).
+
+#DEFINE FILE NAMES
+UNPHASED="${FILE_UNPHASED_PRE}${CHR_NAME}.${FILE_VCF_SUF}"
```

Comment:
Note that the braces around the variable names are usually not necessary in bash. Exceptions are

```bash
"${value:-defaultValue}"
"${possiblyMissingValue:?No value given for possiblyMissingValue}"
"${VARNAME1}_followedByVariableCharacter"
etc.
```